webapp/components/TeaserImage/TeaserImage.vue

```diff
@@ -89,6 +89,7 @@ export default {
     },
     transformImage(file) {
       this.file = file
+      this.$emit('cropInProgress', true)
       this.showCropper = true
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-12-20T19:50:58Z" title="Friday, December 20th 2019, 8:50:58 pm +01:00">Dec 20, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -12,7 +12,7 @@
       <ds-button @click.stop.prevent="cropImage" class="crop-confirm" primary>
         {{ $t('contribution.teaserImage.cropperConfirm') }}
       </ds-button>
-      <ds-button @click="cancelCrop" class="crop-cancel" icon="close"></ds-button>
+      <ds-button @click.prevent="cancelCrop" class="crop-cancel" icon="close"></ds-button>
```


Nice, can we add frontend tests for this behaviour? Or tests in general?

Reply:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2020-01-08T13:13:36Z" title="Wednesday, January 8th 2020, 2:13:36 pm +01:00">Jan 8, 2020</time>_

---


Yes please. Have a Cypress test like @roschaefer suggested.